### PR TITLE
Debugger action is implemented (closes #918)

### LINKED ISSUE #918: Add `t.debugger` action
I want to be able to stop the test to debug:

``` javascript
await t
   .click('body')
   .debugger
   .click('#main_window');
```

After first clicking the test should stand on pause. During the pause, I can take any action in the browser console. To continue the test, I need to press the specifically button ("Continue test..." for example).

### PR DESCRIPTION
**Pull request by helen-dikareva**
/cc @inikulin @AlexanderMoskovkin @AndreyBelym

### CODE REVIEW

**Comment by AlexanderMoskovkin** [comment]
Could you please attach a screenshot or screencast to see how it looks?

**Comment by helen-dikareva** [comment]
debugging

File: src/api/test-controller.js [hunk 1]
```diff
@@ -260,6 +260,10 @@ export default class TestController {
     _expect$ (actual) {
         return new Assertion(actual, this);
     }
+
+    _debugger$ () {
```
Comment (inikulin) [inline src/api/test-controller.js]: It shouldn't be a function, but getter instead:
```js
await t
       .debugger
       .click(...)
        
```

See how it's done here for `t.ctx`: https://github.com/DevExpress/testcafe/blob/master/src/api/test-controller.js#L117 You need to rebase to the latest source.

**Comment by helen-dikareva** [comment]
FPR

File: src/api/test-controller.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import Promise from 'pinkie';
 import { identity, assign, isNil as isNullOrUndefined } from 'lodash';
 import { MissingAwaitError } from '../errors/test-run';
 import getCallsite from '../errors/get-callsite';
-import deprecate from '../warnings/deprecate';
+import showDeprecatedMessage from '../notifications/deprecated-message';
```
Comment (inikulin) [inline src/api/test-controller.js]: `showDeprecationMessage`

File: src/notifications/debugging-message.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import chalk from 'chalk';
+import createStackFilter from '../errors/create-stack-filter';
+
+export default function showDebuggingMessage (callsite, userAgent) {
```
Comment (inikulin) [inline src/notifications/debugging-message.js]: `showDebuggerMessage`

**Comment by AlexanderMoskovkin** [comment]
@helen-dikareva It looks good for me. Could you please add a screenshot of console with the debugger report?

**Comment by helen-dikareva** [comment]
For running test:
```js
fixture `fixture`
.page `http://localhost/testcafe/`;

test('test', async t => {
    await t
        .debug()
        .click('#b1')
        .click('#b2')
        .click('#b3');
});
```
step by step log:
step_by_step

**Comment by helen-dikareva** [comment]
FPR /cc @inikulin @AndreyBelym

**Comment by helen-dikareva** [comment]
Debugger should stop on assertions https://github.com/DevExpress/testcafe/issues/1010

**Pull request merged**